### DISCUSSION TITLE
coins stolen

### DISCUSSION THREAD

**Discussion by kentos8888** [Chia-Network/chia-blockchain#10821] [Support]
everything is simple - they stole coins 03/18/2022 - I discovered it only today 03/21/2022
can anyone help me with the return of the coins?
https://www.chiaexplorer.com/blockchain/address/xch12kkesc4fkuyc964yvg685z4nxzrz2cs4ssw4e34dj4rr9ux46ruq3fj6m8
it is clear that there were no transactions after finding the block
and in the wallet you can see this
then the rest of the coins were scattered at different addresses

![123](https://user-images.githubusercontent.com/102032386/159256283-df16f420-77e8-4a0e-a259-11e97d5f43e3.png)

**Comment by bellinghamy** [comment]
Contact FASTFUNDSRECOVERY@PROTONMAIL to assist you.
